docs/integrations/squid.md

```diff
@@ -1,70 +1,245 @@
 uuid: a0dbb8f3-ca1c-4c6b-aafa-595bd430c0cb
 name: Squid
 
-## Overview
-Squid is a caching proxy for the Web supporting HTTP, HTTPS, FTP, and more. It reduces bandwidth and improves response times by caching and reusing frequently-requested web pages. Squid has extensive access controls and makes a great server accelerator.
+# Squid
+
+Squid is a caching proxy for the Web. It reduces bandwidth and improves response times by caching and reusing frequently-requested web pages. Squid has extensive access controls and makes a great server accelerator. The analysis of squid logs by SEKOIA.IO can detect web threats along as in-progress intrusion and exfiltration attempts.
+
+## Data Sources
+
+Following Table lists the data source offered by this integration.
```

Comment:
```suggestion
Following table lists the data source offered by this integration.
```

Reply:
In some academic papers, the words Table, Figure, Section & co are capitalized. You can have a look on this [blog](https://erinwrightwriting.com/reference-tables-and-figures-in-text/) for an interesting discussion on this. I have no idea if it is strictly related to more industrial English.